### ISSUE TITLE
last slide is not showing first slide on right swipe

### ISSUE THREAD

**Issue by Obsessive** [TheOriginalJosh/nativescript-slides#28]
when loop=true i can move left from first slide to last but i cant move right from last slide to first slide.

The reason is this part of code is changed

```
if (this.loop) {
	slideMap[0].left = slideMap[slideMap.length - 1];
	slideMap[slideMap.length - 1].right = slideMap[0];
}
```

to

```
if (this.loop) {
	slideMap[0].left = slideMap[slideMap.length - 1];
}
```

Is there a reason for doing this?

**Comment by Obsessive** [comment]
pageIndicator is not set to any value but i can see the pageIndicator so i set pageIndicators="false" and this is not hiding the pageIndicator.
I think the problem is with this code
```
if (this._pageIndicators == null) {
	this._pageIndicators = false;
}
```
This should be 
```
if (this._pageIndicators == null) {
	this.pageIndicators = false;
}
```

**Comment by TheOriginalJosh** [comment]
@Obsessive oops, I'm not sure when that part for the loop was removed but that should be added back it as far as i know, thank you for catching this. 

The page indicators shouldn't matter since `this.pageIndicators` and `this._pageIndicators` are essentially the same thing and when you get or set `this.pageIndicators` it's getting or setting `this._pageIndicators`. I'm thinking possibly the version your running or the version on NPM may have a bug? I kinda set an flurry of updates last week and please check that your on the latest version. 

Thanks again!

**Comment by TheOriginalJosh** [comment]
Verified these are fixed in 1.5.2. let me know if you have any more problems.
